### PR TITLE
Update learning.md

### PR DESCRIPTION
**Pull request by samanthasunne**
wrote sql instructions

### CODE REVIEW

File: docs/docs/learning.md [hunk 1]
```diff
@@ -5,5 +5,84 @@ sidebar_label: Learning Resources
...
+## **SQL Queries**
+
+To use SQL, you'll need to understand a few basic concepts and commands. The language uses a certain syntax, or word order, to converse with a database. The key word here is in the middle name of the language: query. In this case, a query is a question written in SQL syntax.
+
+For example, this query asks, "Who has permits for short term rentals (AirBnBs) in New Orleans?"
+
+```
+SELECT contact_name
+FROM str_permits
+WHERE current_status = issued
```
Comment (nihonjinrxs) [inline docs/docs/learning.md]: Does this need quotes?
```suggestion
WHERE current_status = 'issued'
```

File: docs/docs/learning.md [hunk 2]
```diff
@@ -5,5 +5,84 @@ sidebar_label: Learning Resources
...
+```
+
+"SELECT" tells the computer what data you want to see. In this case, we're asking to see the licensees' names. If you want to see all the columns at once, you can type an asterisk "*" to mean "all."
```
Comment (nihonjinrxs) [inline docs/docs/learning.md]: You can use the single back tick to set apart code keywords in monospace typeface, something like this:
```suggestion
`SELECT` tells the computer what data you want to see. In this case, we're asking to see the licensees' names. If you want to see all the columns at once, you can type an asterisk `*` to mean "all."
```
Similar for the ones below too.

File: docs/docs/learning.md [hunk 3]
```diff
@@ -5,5 +5,84 @@ sidebar_label: Learning Resources
...
+"GROUP BY" groups rows into one row if they match somewhere. For instance, it groups all 911 calls for "arson" into the same row.
+
+"ORDER BY" puts the output in a certain order, like alphabetical, chronological, or from smallest to largest.
+
+For example, this query asks for the names of phone numbers of everyone with active permits in New Orleans (or rather, registered in the New Orleans database). But this time, it creates a single row for each person, and gives us the list in alphabetical order.
+
+```
+SELECT contact_name, contact_phone
+FROM str_permits
+WHERE current_status = issued
```
Comment (nihonjinrxs) [inline docs/docs/learning.md]:
```suggestion
WHERE current_status = 'issued'
```

**Review by nihonjinrxs — CHANGES_REQUESTED** [review]
This is a great addition to the nolabase documentation! Thanks so much for your contribution here!

There are a few suggestions here. Most important is that it's important to put the string value in the `WHERE` clauses in single quotes. Other suggestions are for formatting that makes it easier to tell when something is a syntax statement within the prose.

**Comment by samanthasunne** [comment]
Ok I think I added the ` to make the SQL commands stand out. Let me know if they don't show up

**Comment by mrcnc** [comment]
@samanthasunne You can check the[ rendered version from your own branch](https://github.com/samanthasunne/nolabase/blob/patch-1/docs/docs/learning.md).  LGTM 👍

**Comment by samanthasunne** [comment]
Thank you! It's still not appearing on
https://nolabase.codeforneworleans.org/docs/learning/

**Review by nihonjinrxs — APPROVED** [review]
Thanks for the updates!

**Pull request merged**